quicly-probes.d

```diff
@@ -20,6 +20,24 @@
  * IN THE SOFTWARE.
  */
 
+/* @appdata
+{
+    "receive": ["bytes"],
+    "packet_received": ["decrypted"],
+    "stream_on_receive": ["src"],
+    "new_token_send": ["token"],
+    "new_token_receive": ["token"],
+    "new_connection_id_send": ["stateless_reset_token"],
+    "new_connection_id_receive": ["stateless_reset_token"],
```

Comment:
I do not think we need these four. They carry data that are sent on wire (or that might be sent on wire). They aren't application data or they cannot be used to reveal application data.

`new_connection_id_send.statelss_reset_token` exposes information that can be used for disrupting the connection, but I do not think we need to be too worried of that.

Reply:
I agree with @kazuho. I'll add that while `new_token_send` and `new_token_receive` include the token, that piece of information is encrypted. Importantly, the key for the token is only in H2O's memory and so to decrypt it from the logs, an attacker has to access H2O's memory. At which point, the attacker has access to decrypted connections anyway.